pyrefly/lib/lsp/wasm/hover.rs

```diff
@@ -177,6 +194,126 @@ impl HoverValue {
     }
 }
 
+fn collect_typed_dict_fields_for_hover<'a>(
+    solver: &AnswersSolver<TransactionHandle<'a>>,
+    ty: &Type,
+) -> Option<Vec<(Name, Type, Required)>> {
+    match ty {
+        Type::Unpack(inner) => match inner.as_ref() {
+            Type::TypedDict(typed_dict) => {
+                let fields = solver.type_order().typed_dict_kw_param_info(typed_dict);
+                if fields.is_empty() {
+                    None
+                } else {
+                    Some(fields)
+                }
+            }
+            Type::PartialTypedDict(typed_dict) => {
```

Comment:
I don't think partial can appear in a kwargs annotation, it's only produced by local inference

here's its docstring

```
     /// Represents a "partial" version of a TypedDict that can be merged into the TypedDict
    /// (e.g., via its `update` method).
    /// For a TypedDict type `C`, `Partial[C]` represents an object with any subset of read-write
    /// keys from `C`, where each present key has the same value type as in `C`.
```